combine.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# Takes two SVG files (the first two arguments) and layers them above each other.
+# The third argument is the output file.
+# Make sure that the canvas size is the same in all files,
+# otherwise the positioning of the images is not correct!
+
+set -e
```

Comment:
This ensures that the script will terminate if any of the commands executed by it fail.
May be useful for the other script as well...